Appraisals

```diff
@@ -1,5 +1,5 @@
 case(RUBY_VERSION)
-  when '2.3.3', '2.5.3' then
+  when '2.3.3', '2.5.3', '2.6.3' then
```

Comment:
you can drop 2.3.3 here and in the Rakefile

Reply:
Done